### PR TITLE
update `scheduling-hugepages` page

### PR DESCRIPTION
**Pull request by fancc**
update `scheduling-hugepages` page

### CODE REVIEW

**Comment by sftim** [comment]
Please rebase this pull request against _master_ to bring in a work-around for issue #22222.

**Comment by tengqm** [comment]
@fancc please rebase this one onto master branch? it has been here for 2 months.

**Comment by fancc** [comment]
> @fancc please rebase this one onto master branch? it has been here for 2 months.

done

File: content/zh/docs/tasks/manage-hugepages/scheduling-hugepages.md [hunk 1]
```diff
@@ -48,29 +47,70 @@ schedulable resource.
...
+
+A pod may consume multiple huge page sizes in a single pod spec. In this case it
+must use `medium: HugePages-<hugepagesize>` notation for all volume mounts.
 --->
 
-用户可以通过在容器级别的资源需求中使用资源名称 `hugepages-<size>` 来使用巨页，其中的 size 是特定节点上支持的以整数值表示的最小二进制单位。 例如，如果节点支持 2048KiB 的页面规格， 它将暴露可供调度的资源 `hugepages-2Mi`。 与 CPU 或内存不同，巨页不支持过量使用（overcommit）。
-注意，在请求巨页资源时，还必须请求内存或 CPU 资源。
+用户可以通过在容器级别的资源需求中使用资源名称 `hugepages-<size>` 来使用巨页，其中的 size 是特定节点上支持的以整数值表示的最小二进制单位。 例如，如果一个节点支持 2048KiB 和 1048576KiB 页面大小，它将公开可调度的资源 `hugepages-2Mi` 和 `hugepages-1Gi`。与 CPU 或内存不同，巨页不支持过量使用（overcommit）。注意，在请求巨页资源时，还必须请求内存或 CPU 资源。
+
+在单个 pod 的 spec 中，一个 Pod 可能会消耗多个巨页，在这种情况下，它必须对所有 mount 进去的卷使用 `medium: HugePages-<hugepagesize>` 标识。
```
Comment (tengqm) [inline content/zh/docs/tasks/manage-hugepages/scheduling-hugepages.md:63]:
```suggestion
同一 Pod 的 spec 中可能会消耗不同尺寸的巨页。在这种情况下，它必须对所有挂载卷使用 `medium: HugePages-<hugepagesize>` 标识。
```

File: content/zh/docs/tasks/manage-hugepages/scheduling-hugepages.md [hunk 2]
```diff
@@ -48,29 +47,70 @@ schedulable resource.
...
+    emptyDir:
+      medium: HugePages-2Mi
+  - name: hugepage-1gi
+    emptyDir:
+      medium: HugePages-1Gi
+```
+<!--
+A pod may use `medium: HugePages` only if it requests huge pages of one size.
+-->
+Pod 只有在请求一个大小的巨页时才使用 `medium：HugePages`。
```
Comment (tengqm) [inline content/zh/docs/tasks/manage-hugepages/scheduling-hugepages.md:100]:
```suggestion
Pod 只有在请求同一大小的巨页时才使用 `medium：HugePages`。
```

File: content/zh/docs/tasks/manage-hugepages/scheduling-hugepages.md [hunk 3]
```diff
@@ -89,39 +129,33 @@ spec:
...
-- LimitRange support.
---->
-
-## 待实现的特性
-
-- 在 pod 级别隔离的基础上，支持巨页在容器级别的隔离。
-- 作为服务质量特性，保证巨页的 NUMA 局部性。
-- 支持 LimitRange 。
-
+- 特性门控中支持多种尺寸的巨页。它可以通过`HugePageStorageMediumSize` [特性门控](/docs/reference/command-line-tools-reference/feature-gates/)在  {{<
```
Comment (tengqm) [inline content/zh/docs/tasks/manage-hugepages/scheduling-hugepages.md:155]:
```suggestion
- 多种尺寸的巨页的支持需要特性门控配置。它可以通过`HugePageStorageMediumSize` [特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)在  {{<
```

**Pull request merged**